packages/sdk-core/src/model/configuration/BacktraceDatabaseConfiguration.ts

```diff
@@ -0,0 +1,71 @@
+export enum DeduplicationStrategy {
+    None = 0,
+    Callstack = 1 << 0,
+    Classifier = 1 << 1,
+    Message = 1 << 2,
+    All = ~(~0 << 4),
+}
+export type EnabledBacktraceConfiguration = {
```

Comment:
why not `interface`?

Reply:
Pushed adjusted code base on our conversation